python/tests/test_pyhdk_sql.py

```diff
@@ -49,6 +49,15 @@ def execute_sql(cls, sql, **kwargs):
     def get_storage(cls):
         return cls.storage
 
+    def test_null_schema(self):
+        table = pyarrow.table(
+            [[]], schema=pyarrow.schema([pyarrow.field("A", pyarrow.null())])
+        )
+        opt = pyhdk.storage.TableOptions(2)
+        hdk = pyhdk.init()
```

Comment:
This suite initializes individual components and using `HDK` class here through `pyhdk.init` might cause problems due to multiple data managers, etc. created, which is not expected. I propose to create a new test suite dedicated to data import tests.

Reply:
Should I move `TestImport` from `test_pyhdk_api` to `test_pyhdk_data_import`?

Reply:
it can be done separately if we want to, the current version of this PR looks good